sqlmesh/core/model/decorator.py

```diff
@@ -58,6 +58,7 @@ def model(
         dialect: t.Optional[str] = None,
         time_column_format: str = c.DEFAULT_TIME_COLUMN_FORMAT,
         physical_schema_override: t.Optional[t.Dict[str, str]] = None,
+        project: str = "",
```

Comment:
Should this be Optional rather than empty string?

Reply:
i wanted to always be filled do you have a strong pref?

Reply:
Well, empty string is not exactly meaningful, so None is at least explicit. Up to you